bridges/modules/xcm-bridge-hub/src/mock.rs

```diff
@@ -170,16 +170,16 @@ impl pallet_bridge_messages::WeightInfoExt for TestMessagesWeights {
 parameter_types! {
 	pub const RelayNetwork: NetworkId = NetworkId::Kusama;
 	pub const BridgedRelayNetwork: NetworkId = NetworkId::Polkadot;
-	pub const BridgedRelayNetworkLocation: MultiLocation = MultiLocation {
-		parents: 1,
-		interior: X1(GlobalConsensus(BridgedRelayNetwork::get()))
-	};
+	pub BridgedRelayNetworkLocation: Location = Location::new(
+		1,
+		[GlobalConsensus(BridgedRelayNetwork::get())]
```

Comment:
This looks like we can add an implementation of `From<(i32, [Junction; N])> for Location where const N: u32`.